src/components/Table/Icons.tsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+
+export type SortingArrowIcon = React.FC<{
```

Comment:
Can we possibly use `React.FunctionComponent<React.SVGProps<SVGSVGElement>>;` something like this here? Or do you think this might get in our way somehow?

Reply:
i think that's a little too specific, the icon could be different than an svg